plottr/plot/base.py

```diff
@@ -41,7 +41,7 @@ def setPlotWidgetContainer(self, w: 'PlotWidgetContainer'):
         self.plotWidgetContainer = w
         self.newPlotData.connect(self.plotWidgetContainer.setData)
 
-    def process(self, dataIn: DataDictBase = None) -> Dict[str, DataDictBase]:
+    def process(self, dataIn: DataDictBase = None) -> Dict[str, Optional[DataDictBase]]:
```

Comment:
? or is it one of the problematic ones?
```suggestion
    def process(self, dataIn: Optional[DataDictBase] = None) -> Dict[str, Optional[DataDictBase]]:
```